hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java

```diff
@@ -25,6 +25,8 @@
 
     private boolean gravatar;
 
+    private String timezone;
```

Comment:
I would call it `fixedTimezone` or `customTimezone`, in order to show, that we are setting it explicitly